src/entities/addLiquidityNested/index.ts

```diff
@@ -33,6 +34,13 @@ export class AddLiquidityNested {
                 return addLiquidity.query(input, nestedPoolState);
             }
             case 3: {
+                if (input.chainId === ChainId.AVALANCHE) {
+                    throw new SDKError(
+                        'Input Validation',
+                        'Add Liquidity Nested',
+                        'Balancer V3 does not support this operation on Avalanche',
```

Comment:
Mind that consumers will usually show just the error message, so having the actual action within it should be helpful:
```suggestion
                        'Balancer V3 does not support Add Liquidity Nested on Avalanche',
```